### ISSUE TITLE
Adding an existing lesson to a course "removes" the lesson's author

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms#1029]
### Reproduction Steps

+ create a lesson and verify it has an author assigned
+ create a course
+ open the course builder, create a section and add the previously created lesson to this course
+ save
+ check the lesson's author

### Expected Behavior

+ I would expect to still see the same lesson author as before adding the lesson to the course

### Actual Behavior

+ no lesson author anymore

### Error Messages / Logs

```
[25-Feb-2020 20:57:05 UTC] PHP Warning:  strip_tags() expects parameter 1 to be string, array given in /srv/www/wordpress-one/public_html/wp-content/plugins/lifterlms/includes/abstracts/abstract.llms.post.model.php on line 901
```

### System Information

LifterLMS 3.37.10

### Related User Information
HS-110280

Basically, from a rapid view/dump, the lesson's author is an absint but we try to set an array of the type
```
[
    'description' => 
    'email' => 
    'first_name' => 
    'id' => 1
    'last_name' => 
]
```

**Comment by thomasplevy** [comment]
the offending function is here: 

https://github.com/gocodebox/lifterlms/blob/09c3353aaeb2a8785868f2cf1cf67c4146e94068/assets/js/builder/Models/Course.js#L60-L95

We need to replace the object of instructor information with the ID for the duplication, I think.

That should fix it.

**Comment by thomasplevy** [comment]
@eri-trabiccolo  fixed in 3.37.11, please update related HS tickets. Thanks
